lib/ui/screens/types/type_screen.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:pokedex/configs/colors.dart';
+import 'package:pokedex/configs/images.dart';
+import 'package:pokedex/configs/types.dart';
+import 'package:pokedex/routes.dart';
+import 'package:pokedex/ui/screens/types/typeGrid.dart';
+import 'package:pokedex/ui/widgets/poke_container.dart';
+
+class TypeScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: PokeballBackground(buildChildren: (props) {
+        final appBarTop = props.size / 2 + props.top - IconTheme.of(context).size / 2;
+        return [
+          Column(
+            mainAxisSize: MainAxisSize.max,
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: [
+              Padding(
+                padding: EdgeInsets.only(top: appBarTop),
+                child: Stack(
+                  alignment: Alignment.center,
+                  children: <Widget>[
+                    const Positioned(
+                      left: 0,
+                      child: IconButton(
+                        padding: EdgeInsets.all(24),
+                        icon: Icon(Icons.arrow_back),
+                        onPressed: AppNavigator.pop,
+                      ),
+                    ),
+                    const Text(
+                      'Type Effects',
+                      style: TextStyle(
+                        color: Colors.transparent,
+                        fontWeight: FontWeight.bold,
+                        fontSize: 20,
+                      ),
+                    ),
+                    const Positioned(
+                      right: 0,
+                      child: IconButton(
+                        padding: EdgeInsets.all(24),
+                        icon: Icon(Icons.menu),
+                        onPressed: AppNavigator.pop,
+                      ),
+                    ),
+                    // TypeList(),
+                  ],
+                ),
+              ),
+              Padding(
+                padding: EdgeInsets.only(
+                  top: 18,
+                  left: 26,
+                  right: 26,
+                  bottom: 4,
+                ),
+                child: Text(
+                  'Type Effects',
+                  style: TextStyle(
+                    fontSize: 30,
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+              ),
+              TypeGrid()
```

Comment:
The name of `TypeCard, TypeListView, TypeGrid, TypeSheet` is not clear enough and could be used by accident in another screen/widgets. Change it to something like `TypeEffectCard, TypeEffectList, TypeEffectGrid, TypeEffectSheet` instead